src/gui/torrentmodel.h

```diff
@@ -33,15 +34,20 @@
 #define TORRENTMODEL_H
 
 #include <QAbstractListModel>
+#include <QColor>
+#include <QIcon>
+#include <QPixmap>
 #include <QList>
 
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
I was wondering are these function need to be in the class interface. BTW, you may make `TorrentState` an `enum class` with explicit underlying type and then forward declare it, eliminating the include.